Dockerfile

```diff
@@ -20,6 +20,9 @@ FROM base as development
 RUN ["go", "install", "github.com/githubnemo/CompileDaemon@latest"]
 EXPOSE 9012
 
+COPY ./openslides-autoupdate-service /openslides-autoupdate-service
+RUN echo 'replace github.com/OpenSlides/openslides-autoupdate-service => /openslides-autoupdate-service' >> go.mod
```

Comment:
Instead of copying the autoupdate we should just tag a release of the autoupdate service.